### PR TITLE
Add label to service

### PR DESCRIPTION
**Pull request by dmvolod**
Missing to add label to service. This requires to separate service creation in single file operator-service.yaml.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,12 @@ kamel install
 
 This will configure the cluster with the Camel K custom resource definitions and install the operator on the current namespace.
 
+If requires to uninstall Camel K from the OpenShift or Kubernetes, it's nessesary to run following command using "oc" or "kubectl" tool:
```
Comment (nicolaferraro) [inline README.md]: Maybe better moving this to the bottom, in a specific paragraph about uninstallation

File: deploy/operator-service.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```
Comment (nicolaferraro) [inline deploy/operator-service.yaml]: Ok, is this required for the metrics feature of the operator-sdk? I don't recall why it's missing

File: deploy/resources.go [hunk 3]
```diff
@@ -86,7 +86,6 @@ spec:
     metadata:
       labels:
         name: camel-k-operator
-        app: "camel-k"
```
Comment (nicolaferraro) [inline deploy/resources.go]: Maybe this is out-of-sync. I've added a `make build-embed-resources` to sync that file. We should keep "camel-k" in the app label for the infrastructure components.

File: deploy/operator-service.yaml (hunk 2 above)
Comment (nicolaferraro) [reply deploy/operator-service.yaml]: I mean the service

File: deploy/resources.go (hunk 3 above)
Comment (nicolaferraro) [reply deploy/resources.go]: Found it. It's the operator sdk that changes it. We can leave it and the source file without the label, as the file is not used (for this reason).

File: deploy/operator-service.yaml (hunk 2 above)
Comment (dmvolod) [reply deploy/operator-service.yaml]: Yes, this is for metrics. It creates by default together with deployment but it's unable to control it parameters there. This is reason why we create it separately.

File: README.md (hunk 1 above)
Comment (dmvolod) [reply README.md]: Moved it.

**Pull request merged**